src/DssSpell.t.sol

```diff
@@ -393,15 +383,15 @@ contract DssSpellTest is DssSpellTestBase {
         assertTrue(spell.done());
 
         // Insert new ilk registry values tests here
-        // RWA014
-        assertEq(reg.pos("RWA014-A"),    61);
-        assertEq(reg.join("RWA014-A"),   addr.addr("MCD_JOIN_RWA014_A"));
-        assertEq(reg.gem("RWA014-A"),    addr.addr("RWA014"));
-        assertEq(reg.dec("RWA014-A"),    GemAbstract(addr.addr("RWA014")).decimals());
-        assertEq(reg.class("RWA014-A"),  3);
-        assertEq(reg.pip("RWA014-A"),    addr.addr("PIP_RWA014"));
-        assertEq(reg.name("RWA014-A"),   "RWA014-A: Coinbase Custody");
-        assertEq(reg.symbol("RWA014-A"), GemAbstract(addr.addr("RWA014")).symbol());
+        // RWA015
+        assertEq(reg.pos("RWA015-A"),    62);
+        assertEq(reg.join("RWA015-A"),   addr.addr("MCD_JOIN_RWA015_A"));
+        assertEq(reg.gem("RWA015-A"),    addr.addr("RWA015"));
+        assertEq(reg.dec("RWA015-A"),    GemAbstract(addr.addr("RWA015")).decimals());
+        assertEq(reg.class("RWA015-A"),  3);
+        assertEq(reg.pip("RWA015-A"),    addr.addr("PIP_RWA015"));
```

Comment:
`pipRwaXXX` has previously been fetched from the oracle itself rather than using the hardcoded value our addresses.

Please see RWA008 and 009 for examples of this:

`// RWA009
        (, address pipRwa009,,) = oracle.ilks("RWA009-A");`

This was done even though the `addr.addr(` pattern was used in that test for 008 and 009, so there was likely a reason for it to be done this way (although I am not immediately seeing a drawback with the current method).

What do you think? I don't see any risk to checking against our addresses file as we have precomputed, but if it was done this way in the past there's probably a reason (unless the test was improved and it was a mistake).